File: java/leetcode/Problem2413.java
package leetcode;

/**
 * 题目链接：https://leetcode.cn/problems/smallest-even-multiple/
 */
class Problem2413 {
    public int smallestEvenMultiple(int n) {
        if (n % 2 == 0) {
            return n;
        }
        return 2 * n;
    }
    public static void main(String[] args) {
        Problem2413 problem = new Problem2413();
        int result = problem.smallestEvenMultiple(5);
        System.out.println(result);
    }
}

File: go/leetcode/problem_2413.go
package leetcode

// 题目链接：https://leetcode.cn/problems/smallest-even-multiple/
func smallestEvenMultiple(n int) int {
    if n % 2 == 0 {  // 是偶数
        return n
    }
    // 比写else快很多
    return n + n
}